### PR TITLE
docker build option

### PR DESCRIPTION
**Pull request by rwcarlsen**
replaces #169

### CODE REVIEW

File: docker/README.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+``fuelcycle.org-deps`` contains a dockerfile with all dependencies for
+building cymetric.  This is used by the docker based build targets added to
```
Comment (gidden) [inline docker/README.rst]: cymetric typo, perhaps there are others?

**Comment by gidden** [comment]
a little doc typo then I think we're good to go. I assume you want the circle PR to be separate?

**Comment by rwcarlsen** [comment]
Ready for a merge.

**Comment by rwcarlsen** [comment]
@mbmcgarry Could you try installing boot2docker (https://docs.docker.com/installation/mac/) and then running ``make html-docker`` to build the website and see if that works for you?

**Comment by gidden** [comment]
I'm in the process of trying this on my mac partition

**Comment by gidden** [comment]
it works and that is beautiful

**Comment by mbmcgarry** [comment]
It works for me too!  Awesome!

-Meghan

On Aug 5, 2015, at 8:02 AM, Robert Carlsen <notifications@github.com<mailto:notifications@github.com>> wrote:


@mbmcgarry<https://github.com/mbmcgarry> Could you try installing boot2docker (https://docs.docker.com/installation/mac/) and then running make html-docker to build the website and see if that works for you?

**Pull request merged**